scripts/aexpect_helper

```diff
@@ -130,7 +130,6 @@ def main():     # too-many-* pylint:disable=R0914,R0912,R0915
 
             # Read from child and write to files/pipes
             server_log.info('Entering main read loop')
-            status = 1  # lgtm [py/multiple-definition]
```

Comment:
OK, so line 183 will only be reached when a valid `status` is set on line 172. LGTM.